lib/open_food_network/feature_toggle.rb

```diff
@@ -10,9 +10,6 @@ module FeatureToggle
     # Please add your new feature here to appear in the Flipper UI.
     # We way move this to a YAML file when it becomes too awkward.
     CURRENT_FEATURES = {
-      "admin_style_v2" => <<~DESC,
-        Change some colour and layout in the backend to a newer version.
-      DESC
```

Comment:
This does not delete the feature from the database. My idea was to move this to `RETIRED_FEATURES` below and then it will be deleted automatically.

I was also thinking of changing this to delete all features that are not in this list but I needed to wait until everyone was using this list to put their existing features in. I think that's the case now.

Reply:
Yes, I didn't think about how it would get deleted, if only I'd read to the bottom of that file!

I agree about just deleting all feature that are not in CURRENT_FEATURES 👍

Reply:
Resolved in https://github.com/openfoodfoundation/openfoodnetwork/pull/10792